res/database/Default/user/therob.xml

```diff
@@ -2346,19 +2651,23 @@
 			<title>
 				<de>%birth% %station%</de>
 				<en>%birth% of %station%</en>
+				<pl>%birth% %station%</pl>
 			</title>
 			<description>
 				<de>Sie haben die Eröffnungsshow verpasst? Hier sind die besten Szenen, Outtakes und Erinnerungen der Teilnehmer.</de>
 				<en>You missed the opening show? Here are the best scenes and outtakes, as well as some retrospections from participants.</en>
+				<pl>Przegapiłeś show otwarcia stacji? Oto najlepsze sceny i urywki, a także kilka retrospekcji od uczestników.</pl>
 			</description>
 			<variables>
 				<birth>
 					<de>Geburt|Die Geburtsstunde|Der Ursprung|Die Entstehung|Die Wurzeln|Die Eröffnung|Der Kick-off|Der Start|Der Anfang|Die Taufe|Die Genese|Die Einweihung|Die Einführung|Die Gründung|Die Morgendämmerung|Das Auftauchen|Das Erscheinen|Die Ankunft</de>
 					<en>Birth|The First Hours|The Origin|The Formation|The Roots|The Opening|The Launch|The Beginning|The Inception|The Onset|The Genesis|The Inauguration|The Initiation|The Founding|The Dawning|The Emergence|The Appearance|The Arrival</en>
+					<pl>Narodziny|Pierwsze godziny|Początek|Formowanie|Stworzenie|Otwarcie|Uruchomienie|Początek|Początek|Geneza|Inauguracja|Inicjacja|Założenie|Powstanie|Start|Pojawienie się|Utworzenie</pl>
```

Comment:
different number of alternatives will cause problems

Reply:
I don't know why I overlooked. corrected.